docs/docsite/rst/dev_guide/testing_integration.rst

```diff
@@ -25,6 +25,22 @@ It provides tab completion in ``bash`` for the ``ansible-test`` test runner.
 Configuration
 =============
 
+ansible-test command
+--------------------
+
+The example below assumes ``bin/`` is in your ``$PATH``. An easy way to achieve that
+is to initialize your environment with the ``env-setup`` command::
+
+    source hacking/env-setup
+    ansible-test --help
+
+You can also call ``ansible-test`` with the full path::
+
+    bin/ansible-test --help
+
+integration_config.yml
+----------------------
+
```

Comment:
The section below that references `credentials.template` is out-of-date as well. I'd probably replace the last sentence:

> For more information about supported credentials, refer to ``credentials.template``.

with:

> For more information about supported credentials, refer to the various ``cloud-config-*.template`` files in the ``test/integration/`` directory.